dbt_coding_conventions.md

```diff
@@ -22,18 +16,29 @@
 }}
 ```
 
-## Base Models
 
-- Only base models should select from source tables / views
-- Only a single base model should be able to select from a given source table / view.
-- Base models should be placed in a base/ directory
-- Base models should perform all necessary data type casting
-- Base models should perform all field naming to force field names to conform to standard field naming conventions
-- Source fields that use reserved words must be renamed in base models
+## dbt conventions
+* Only `stg_` models (or `base_` models if your project requires them) should select from `source`s.
+* All other models should only select from other models.
+* For more information on how we structure our projects see [this document](link to be created)
+
+## Testing
+
+- Every subdirectory should contain a `schema.yml` file, in which each model in the subdirectory is tested.
+- At a minimum, unique and foreign key constraints should be tested (if applicable).

```

Comment:
IMO it should be unique and not-null on PKs. I don't actually like testing FKs because there are too often failures when ingestion schedules are not perfectly synchronized between multiple different sources. This introduces errors that are not errors.

Reply:
Agreed, brought this over from the old coding conventions. Will update